app/api/helpers/csv_jobs_util.py

```diff
@@ -82,75 +82,127 @@ def export_orders_csv(orders):
     return rows
 
 
-def export_attendees_csv(attendees, custom_forms, attendee_form_dict):
-    return_dict_list = []
+def get_order_ticket_data(order, ticket):
+    """Get order ticket data"""
+    data = {}
+    if not order:
+        return {
+            'Order#': '',
+            'Order Date': '',
+            'Status': '',
+            'Payment Type': '',
+            'Payment Mode': '',
+            'Ticket ID': '',
+            'Ticket Name': '',
+            'Ticket Price': '0',
+            'Ticket Type': '',
+            'Tax ID': '',
+            'Address': '',
+            'Company': '',
+            'Country': '',
+            'State': '',
+            'City': '',
+            'Zipcode': '',
+        }
 
-    for attendee in attendees:
-        data = {
-            'Order#': str(attendee.order.get_invoice_number()) if attendee.order else '-',
-            'Order Date': str(attendee.order.created_at.strftime('%B %-d, %Y %H:%M %z'))
-            if attendee.order and attendee.order.created_at
-            else '-',
-            'Status': str(attendee.order.status)
-            if attendee.order and attendee.order.status
-            else '-',
-            'Payment Type': str(attendee.order.paid_via)
-            if attendee.order and attendee.order.paid_via
-            else '',
-            'Payment Mode': str(attendee.order.payment_mode)
-            if attendee.order and attendee.order.payment_mode
-            else '',
-            'Ticket Name': str(attendee.ticket.name)
-            if attendee.ticket and attendee.ticket.name
-            else '',
-            'Ticket Price': str(attendee.ticket.price)
-            if attendee.ticket and attendee.ticket.price
-            else '0',
-            'Ticket Type': str(attendee.ticket.type)
-            if attendee.ticket and attendee.ticket.type
-            else '',
-            'Tax ID': str(attendee.order.tax_business_info)
-            if attendee.order.tax_business_info
-            else '',
-            'Address': str(attendee.order.address) if attendee.order.address else '',
-            'Company': str(attendee.order.company) if attendee.order.company else '',
-            'Country': str(attendee.order.country) if attendee.order.country else '',
-            'State': str(attendee.order.state) if attendee.order.state else '',
-            'City': str(attendee.order.city) if attendee.order.city else '',
-            'Zipcode': str(attendee.order.zipcode) if attendee.order.zipcode else '',
-            'Email': '',
+    data = {
+        'Order#': str(order.get_invoice_number()),
```

Comment:
order number and not invoice number and line 58 too

Reply:
Hello @mariobehling, can you share about this idea?

Reply:
Every order has a relation to tickets and invoice. Each has different number. 

{
    "data": {
        "type": "order",
        "attributes": {
            "order-notes": null,
            **"identifier": "ad2ad60f-33f1-456e-9fec-7d0c3151f217",**
            "created-at": "2024-03-20T03:09:04.041344+00:00",
            "tax-business-info": null,
            "exp-month": null,
            "address": null,
            "tickets-pdf-url": "https://api.eventyay.com/generated/tickets/orders/tickets/pdf/ad2ad60f-33f1-456e-9fec-7d0c3151f217/ad2ad60f-33f1-456e-9fec-7d0c3151f217/b1pBQ1dDQ0/ad2ad60f-33f1-456e-9fec-7d0c3151f217.pdf",
            "transaction-id": null,
            "paid-via": null,
            "state": null,
            "company": null,
            "city": null,
            "completed-at": "2024-03-20T03:10:41.185012+00:00",
            "discount-code-id": "71117",
            "amount": 0.0,
            "brand": null,
            "status": "completed",
            "country": null,
            "exp-year": null,
            "last4": null,
            "is-billing-enabled": false,
            "cancel-note": null,
            "zipcode": null,
            "payment-mode": "free"
        },
        "relationships": {
            "marketer": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/marketer"
                }
            },
            "tickets": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/ticket",
                    "related": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/tickets"
                },
                "data": [
                    {
                        "type": "ticket",
                        "id": "3458"
                    },
                    {
                        "type": "ticket",
                        "id": "3464"
                    }
                ]
            },
            "discount-code": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/discount-code",
                    "related": "/v1/discount-codes/71117"
                }
            },
            "event": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/event",
                    "related": "/v1/events/3284"
                },
                "data": {
                    "type": "event",
                    "id": "3284"
                }
            },
            "user": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/user",
                    "related": "/v1/users/33522"
                }
            },
            "attendees": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/attendee",
                    "related": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/attendees"
                },
                "data": [
                    {
                        "type": "attendee",
                        "id": "49414"
                    },
                    {
                        "type": "attendee",
                        "id": "49413"
                    }
                ]
            },
            "event-invoice": {
                "links": {
                    "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217/relationships/event-invoice/",
                    **"related": "/v1/event-invoices/38205"**
                }
            }
        },
        "id": "38205",
        "links": {
            "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217"
        }
    },
    "links": {
        "self": "/v1/orders/ad2ad60f-33f1-456e-9fec-7d0c3151f217"
    },
    "included": [
        {
            "type": "ticket",
            "attributes": {
                "is-fee-absorbed": true,
                "max-price": 0.0,
                "is-hidden": false,
                "quantity": 1000,
                "sales-ends-at": "2024-04-10T11:00:00+00:00",
                "min-price": 1.0,
                "is-checkin-restricted": true,
                "is-description-visible": true,
                "min-order": 1,
                "type": "free",
                "badge-id": "",
                "price": null,
                "name": "Free Virtual Ticket (V\u00e9 xem tr\u1ef1c tuy\u1ebfn)",
                "position": 7,
                "auto-checkin-enabled": false,
                "deleted-at": null,
                "description": "Access to all sessions online via event platform",
                "sales-starts-at": "2023-10-08T15:31:40.051000+00:00",
                "max-order": 1,
                "form-id": "e313b531-1028-46f3-8c74-ab26da5cb240"
            },
            "relationships": {
                "ticket-tags": {
                    "links": {
                        "self": "/v1/tickets/3458/relationships/ticket-tags",
                        "related": "/v1/tickets/3458/ticket-tags"
                    }
                },
                "discount-codes": {
                    "links": {
                        "self": "/v1/tickets/3458/relationships/discount-codes",
                        "related": "/v1/tickets/3458/discount-codes"
                    }
                },
                "event": {
                    "links": {
                        "self": "/v1/tickets/3458/relationships/event",
                        "related": "/v1/tickets/3458/event"
                    }
                }
            },
            "id": "3458",
            "links": {
                "self": "/v1/tickets/3458"
            }
        },
        {
            "type": "ticket",
            "attributes": {
                "is-fee-absorbed": true,
                "max-price": 0.0,
                "is-hidden": false,
                "quantity": 8000,
                "sales-ends-at": "2024-04-10T11:00:00+00:00",
                "min-price": 1.0,
                "is-checkin-restricted": true,
                "is-description-visible": true,
                "min-order": 1,
                "type": "paid",
                "badge-id": "",
                "price": 30.0,
                "name": "Individual Ticket (V\u00e9 C\u00e1 Nh\u00e2n) ",
                "position": 4,
                "auto-checkin-enabled": false,
                "deleted-at": null,
                "description": "Full access to the conference including catering. ",
                "sales-starts-at": "2023-10-08T15:31:37.390000+00:00",
                "max-order": 1,
                "form-id": "e313b531-1028-46f3-8c74-ab26da5cb240"
            },
            "relationships": {
                "ticket-tags": {
                    "links": {
                        "self": "/v1/tickets/3464/relationships/ticket-tags",
                        "related": "/v1/tickets/3464/ticket-tags"
                    }
                },
                "discount-codes": {
                    "links": {
                        "self": "/v1/tickets/3464/relationships/discount-codes",
                        "related": "/v1/tickets/3464/discount-codes"
                    }
                },
                "event": {
                    "links": {
                        "self": "/v1/tickets/3464/relationships/event",
                        "related": "/v1/tickets/3464/event"
                    }
                }
            },
            "id": "3464",
            "links": {
                "self": "/v1/tickets/3464"
            }
        },
        {
            "type": "event",
            "relationships": {
                "event-topic": {
                    "links": {
                        "self": "/v1/events/3284/relationships/event-topic",
                        "related": "/v1/events/3284/event-topic"
                    }
                },
                "badge-forms": {
                    "links": {
                        "self": "/v1/events/3284/relationships/badge-forms",
                        "related": "/v1/events/3284/badge-forms"
                    }
                },
                "event-type": {
                    "links": {
                        "self": "/v1/events/3284/relationships/event-type",
                        "related": "/v1/events/3284/event-type"
                    }
                },
                "group": {
                    "links": {
                        "self": "/v1/events/3284/relationships/group",
                        "related": "/v1/events/3284/group"
                    }
                },
                "ticket-tags": {
                    "links": {
                        "self": "/v1/events/3284/relationships/ticket-tags",
                        "related": "/v1/events/3284/ticket-tags"
                    }
                },
                "order-statistics": {
                    "links": {
                        "self": "/v1/events/3284/relationships/order-statistics",
                        "related": "/v1/events/3284/order-statistics"
                    }
                },
                "sponsors": {
                    "links": {
                        "self": "/v1/events/3284/relationships/sponsors",
                        "related": "/v1/events/3284/sponsors"
                    }
                },
                "organizers": {
                    "links": {
                        "self": "/v1/events/3284/relationships/organizers",
                        "related": "/v1/users"
                    }
                },
                "speakers-call": {
                    "links": {
                        "self": "/v1/events/3284/relationships/speakers-call",
                        "related": "/v1/events/3284/speakers-call"
                    }
                },
                "tax": {
                    "links": {
                        "self": "/v1/events/3284/relationships/tax",
                        "related": "/v1/events/3284/tax"
                    }
                },
                "station": {
                    "links": {
                        "self": "/v1/events/3284/relationships/station",
                        "related": "/v1/events/3284/stations"
                    }
                },
                "speaker-invites": {
                    "links": {
                        "self": "/v1/events/3284/relationships/speaker-invites",
                        "related": "/v1/events/3284/speaker-invites"
                    }
                },
                "speakers": {
                    "links": {
                        "self": "/v1/events/3284/relationships/speakers",
                        "related": "/v1/events/3284/speakers"
                    }
                },
                "tracks": {
                    "links": {
                        "self": "/v1/events/3284/relationships/tracks",
                        "related": "/v1/events/3284/tracks"
                    }
                },
                "exhibitors": {
                    "links": {
                        "self": "/v1/events/3284/relationships/exhibitors",
                        "related": "/v1/events/3284/exhibitors"
                    }
                },
                "tickets": {
                    "links": {
                        "self": "/v1/events/3284/relationships/tickets",
                        "related": "/v1/events/3284/tickets"
                    }
                },
                "video-stream": {
                    "links": {
                        "self": "/v1/video-streams/3284/relationships/event",
                        "related": "/v1/events/3284/video-stream"
                    }
                },
                "session-favourites": {
                    "links": {
                        "related": "/v1/events/3284/user-favourite-sessions"
                    }
                },
                "owner": {
                    "links": {
                        "self": "/v1/events/3284/relationships/owner",
                        "related": "/v1/events/3284/owner"
                    }
                },
                "coorganizers": {
                    "links": {
                        "self": "/v1/events/3284/relationships/coorganizers",
                        "related": "/v1/users"
                    }
                },
                "sessions": {
                    "links": {
                        "self": "/v1/events/3284/relationships/sessions",
                        "related": "/v1/events/3284/sessions"
                    }
                },
                "microlocations": {
                    "links": {
                        "self": "/v1/events/3284/relationships/microlocations",
                        "related": "/v1/events/3284/microlocations"
                    }
                },
                "general-statistics": {
                    "links": {
                        "self": "/v1/events/3284/relationships/general-statistics",
                        "related": "/v1/events/3284/general-statistics"
                    }
                },
                "faq-types": {
                    "links": {
                        "self": "/v1/events/3284/relationships/faq-types",
                        "related": "/v1/events/3284/faq-types"
                    }
                },
                "faqs": {
                    "links": {
                        "self": "/v1/events/3284/relationships/faqs",
                        "related": "/v1/events/3284/faqs"
                    }
                },
                "social-links": {
                    "links": {
                        "self": "/v1/events/3284/relationships/social-links",
                        "related": "/v1/events/3284/social-links"
                    }
                },
                "event-sub-topic": {
                    "links": {
                        "self": "/v1/events/3284/relationships/event-sub-topic",
                        "related": "/v1/events/3284/event-sub-topic"
                    }
                },
                "session-types": {
                    "links": {
                        "self": "/v1/events/3284/relationships/session-types",
                        "related": "/v1/events/3284/session-types"
                    }
                },
                "event-copyright": {
                    "links": {
                        "self": "/v1/events/3284/relationships/event-copyright",
                        "related": "/v1/events/3284/event-copyright"
                    }
                },
                "stripe-authorization": {
                    "links": {
                        "self": "/v1/stripe-authorizations/3284/relationships/event",
                        "related": "/v1/events/3284/stripe-authorization"
                    }
                },
                "feedbacks": {
                    "links": {
                        "self": "/v1/events/3284/relationships/feedbacks",
                        "related": "/v1/events/3284/feedbacks"
                    }
                },
                "custom-forms": {
                    "links": {
                        "self": "/v1/events/3284/relationships/custom-forms",
                        "related": "/v1/events/3284/custom-forms"
                    }
                }
            },
            "attributes": {
                "created-at": "2023-09-19T10:10:54.071894+00:00",
                "is-sponsors-enabled": true,
                "is-promoted": true,
                "logo-url": "https://api.eventyay.com/static/media/temp/images/d71dbba5-f7c7-470a-9754-88bc309df545/MXErRGt2bj/24858afa-6b9c-4013-81da-721a48be5074.png",
                "icon-image-url": "https://api.eventyay.com/static/media/events/3284/icon/SzR0a3N5Vn/92cfaec0-30a5-4131-8aa2-06ca54f67137.jpg",
                "can-pay-by-alipay": false,
                "can-pay-by-cheque": false,
                "is-stripe-linked": true,
                "bank-details": "B\u1ea1n c\u00f3 th\u1ec3 chuy\u1ec3n kho\u1ea3n v\u00e0o TK sau, n\u1ed9i dung chuy\u1ebfn kho\u1ea3n: Vui l\u00f2ng \u0111i\u1ec1n \u0111ia ch\u1ec9 email c\u1ee7a b\u1ea1n: \n\nC\u00d4NG TY TNHH \u0110\u00c0O T\u1ea0O EDEN\nS\u1ed1 TK: 19037239984888\nNg\u00e2n H\u00e0ng: Techcombank",
                "show-remaining-tickets": false,
                "can-pay-by-omise": false,
                "can-pay-by-stripe": true,
                "is-chat-enabled": false,
                "can-pay-by-paytm": false,
                "can-pay-by-paypal": true,
                "stream-loop": false,
                "is-videoroom-enabled": false,
                "can-pay-onsite": false,
                "description": "<p></p><p></p><p>FOSSASIA Summit 2024 s\u1ebd di\u1ec5n ra v\u00e0o April 8-10 at  H\u1ecdc Vi\u1ec7n C\u00f4ng Ngh\u1ec7 B\u01b0u Ch\u00ednh Vi\u1ec5n Th\u00f4ng Hanoi, Vietnam. Join 3,000 developers discussing Cloud, AI, Robotics, Web3, Cybersecurity, Database, Operation System, Hardware, and Semiconductor (C\u00f4ng Ngh\u1ec7 B\u00e1n D\u1eabn). Secure your spot today! </p><p>HIGHLIGHTS</p><ul><li><p><strong>Web3 Builders\u2019 Day:</strong> Featuring Etherum and Arbitrum (a layer two ET solution). Don\u2019t miss a chance to learn how to build and launch  Web3 projects on the world\u2019s largest programmable blockchain. </p></li></ul><ul><li><p><strong>Database:</strong> <a href=\"https://2024.fossasia.org/pgday/\" rel=\"nofollow\" target=\"_blank\">PostgreSQL Day</a> showcasing how to maximise system\u2019s efficiency and performance</p></li></ul><ul><li><p><strong>Operating System:</strong> Featuring openEuler - an Asian\u2019s born open source operating system for digital infrastructure (H\u1ec7 \u0111i\u1ec1u h\u00e0nh th\u00fac \u0111\u1ea9y chuy\u1ec3n \u0111\u1ed5i s\u1ed1 v\u00e0 kinh t\u1ebf s\u1ed1) </p></li></ul><ul><li><p><strong>AI &amp; Cloud:</strong> OpenAI, GenAI, LLMs, K8s, CI/CD, OpenTelemetry, WebAssembly</p></li></ul><ul><li><strong>Cybersecurity</strong>: Passwordless, Zero Trust, Privacy with TOR, Supply Chain Security, Network Security, Quantum Threats</li></ul><ul><li><p><strong>Open Source Hardware and Semiconductor:</strong> RISC-V gi\u1ea3i ph\u00e1p c\u00f4ng ngh\u1ec7 b\u00e1n d\u1eabn ngu\u1ed3n m\u1edf cho thi\u1ebft k\u1ebf chip v\u00e0 h\u1ec7 th\u1ed1ng t\u00edch h\u1ee3p, open source firmware development</p></li></ul><strong></strong>EXHIBITION &amp; NETWORKING<br><ul><li>Tri\u1ec5n l\u00e3m CNTT,  Recruitment Opportunities, Business Matching, Social Events and B\u1ed1c Th\u0103m Tr\u00fang Th\u01b0\u1edfng h\u1eb1ng ng\u00e0y</li></ul>\u0110\u0102NG K\u00dd NGAY!<br><ul><li><p>V\u00e9 \u0111\u00e3 bao g\u1ed3m \u0103n u\u1ed1ng, ch\u1ecdn Schedule \u0111\u1ec3 xem ch\u01b0\u01a1ng tr\u00ecnh c\u1ee5 th\u1ec3</p></li></ul><ul><li><p>H\u01b0\u1edbng d\u1eabn \u0111\u1eb7t v\u00e9: <a href=\"https://youtu.be/blyijO6w9Nc?si=33Lxa4WTA3cwZK0q\" rel=\"nofollow\" target=\"_blank\">https://youtu.be/blyijO6w9Nc?si=33Lxa4WTA3cwZK0q</a></p></li></ul>",
                "owner-name": "",
                "location-name": "Posts and Telecommunications Institute of Technology, Km 10, Ng\u00f5 5 Tr\u1ea7n Ph\u00fa, Ph\u01b0\u1eddng M\u1ed7 Lao, Ha Dong District, H\u00e0 N\u1ed9i, 12011, Vietnam",
                "external-event-url": null,
                "longitude": 0.0,
                "is-cfs-enabled": true,
                "public-stream-link": null,
                "is-tax-enabled": false,
                "ticket-url": null,
                "payment-currency": "USD",
                "is-sessions-speakers-enabled": true,
                "starts-at": "2024-04-08T02:00:00+00:00",
                "refund-policy": "Please note that once your order is completed, it is no longer possible to cancel or refund the purchase.",
                "paypal-email": "office@fossasia.org",
                "is-document-enabled": false,
                "is-badges-enabled": false,
                "deleted-at": null,
                "stream-autoplay": false,
                "timezone": "Asia/Ho_Chi_Minh",
                "payment-country": "Singapore",
                "code-of-conduct": "<p>A member of the FOSSASIA community is:</p><p></p>Open<br>Members of the community are open to collaboration, whether it's on PEPs, patches, problems, or otherwise. We're receptive to constructive comment and criticism, as the experiences and skill sets of other members contribute to the whole of our efforts. We're accepting of all who wish to take part in our activities, fostering an environment where anyone can participate and everyone can make a difference.<br><br>Considerate<br>Members of the community are considerate of their peers -- other FOSSASIA users. We're thoughtful when addressing the efforts of others, keeping in mind that often times the labor was completed simply for the good of the community. We're attentive in our communications, whether in person or online, and we're tactful when approaching differing views.<br><br>Respectful<br>Members of the community are respectful. We're respectful of others, their positions, their skills, their commitments, and their efforts. We're respectful of the volunteer efforts that permeate the FOSSASIA community. We're respectful of the processes set forth in the community, and we work within them. When we disagree, we are courteous in raising our issues.<br><br>Overall, we're good to each other. We contribute to this community not because we have to, but because we want to. If we remember that, these guidelines will come naturally.<br> <br>Note<br><br>This document is based on Code of Conduct of the Python Software Foundation: <a href=\"https://www.python.org/psf/codeofconduct/\" rel=\"nofollow\" target=\"_blank\">https://www.python.org/psf/codeofconduct/</a><br> <br><strong>Anti-Harassment Policy<br></strong><br>Simply having an anti-harassment policy can prevent harassment all by itself and encourages people to attend who have had bad experiences at other events. It also gives event staff instructions on how to handle harassment quickly, with the minimum amount of disruption or bad press for the event. <br><br>FOSSASIA is dedicated to providing harassment-free experience at our events for everyone, regardless of gender, sexual orientation, disability, gender identity, age, race, or religion. We do not tolerate harassment of event participants in any form. Participants asked to stop any harassing behavior are expected to comply immediately. Exhibiting partners at events are also subject to the anti-harassment policy. In particular, exhibitors should not use sexualized images, activities, or other material, or otherwise create a sexualized environment.<br><br>If a participant engages in harassing behavior, the event organizers may take any action they deem appropriate, including warning the offender or expulsion from the event. If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact a member of conference staff immediately. Our Event Staff can usually be identified by special badges/attire.<br><br>Please note, while we take all concerns raised seriously, we will use our discretion as to in determining when and how to follow up on reported incidents and may decline to take any further action and/or may direct the participant to other resources for resolution. Event staff will be happy to help participants contact venue/event security or local law enforcement, provide escorts, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.<br>We expect participants to follow these rules at all event venues and related social events.<br><br>License and attributionThis policy is licensed under the Creative Commons Zero license.",
                "pentabarf-url": "https://api.eventyay.com/static/media/exports/3284/pentabarf/eVhia1RPWV/pentabarf.xml",
                "document-links": null,
                "is-map-shown": true,
                "is-billing-info-mandatory": false,
                "is-featured": true,
                "identifier": "55d2a466",
                "latitude": 0.0,
                "is-oneclick-signup-enabled": false,
                "thumbnail-image-url": "https://api.eventyay.com/static/media/events/3284/thumbnail/akRTSjFLNV/3a9b54a0-a30c-4c63-97b5-8bc468830cb6.jpg",
                "chat-room-name": "FOSSASIA-Summit-2024-55d2a466",
                "is-donation-enabled": false,
                "ical-url": "https://api.eventyay.com/static/media/exports/3284/ical/RjBIYW1ldz/event_ical.ics",
                "invoice-details": null,
                "name": "FOSSASIA Summit 2024",
                "after-order-message": "Thank you for your order. See you at the event!",
                "can-pay-by-bank": true,
                "original-image-url": "https://api.eventyay.com/static/media/temp/images/2e794f4b-b450-4416-aa2e-ff2967b1ebe1/eDBpTzlFOT/c92b658f-3845-4683-aefe-751c351d4f38.jpeg",
                "can-pay-by-invoice": false,
                "online": false,
                "schedule-published-on": "2024-03-01T08:59:41.308000+00:00",
                "is-announced": false,
                "cheque-details": null,
                "has-owner-info": false,
                "large-image-url": "https://api.eventyay.com/static/media/events/3284/large/cXE4aERIQX/6e457983-b121-4882-a755-fdde25bdcab2.jpg",
                "searchable-location-name": null,
                "is-ticket-form-enabled": true,
                "ends-at": "2024-04-10T11:00:00+00:00",
                "state": "published",
                "privacy": "public",
                "is-demoted": false,
                "xcal-url": "https://api.eventyay.com/static/media/exports/3284/xcal/bmh6eWlrdW/xcal.xcs",
                "owner-description": "",
                "onsite-details": null
            },
            "id": "3284",
            "links": {
                "self": "/v1/events/3284"
            }
        },
        {
            "type": "attendee",
            "attributes": {
                "accept-video-recording": false,
                "ticket-id": "3464",
                "lastname": "Trinh Van",
                "checkin-times": null,
                "shipping-address": null,
                "complex-field-values": null,
                "checkout-times": null,
                "twitter": null,
                "tax-business-info": null,
                "website": null,
                "home-address": null,
                "badge-printed-at": null,
                "work-phone": null,
                "billing-address": null,
                "email": "trinhvandat90399@gmail.com",
                "address": null,
                "phone": null,
                "is-consent-of-refund-policy": false,
                "linkedin": null,
                "register-times": null,
                "native-language": null,
                "home-wiki": null,
                "job-title": "Java backend",
                "is-consent-form-field": false,
                "pdf-url": "https://api.eventyay.com/generated/tickets/orders/tickets/pdf/ad2ad60f-33f1-456e-9fec-7d0c3151f217/49414/Q3dXdjV0OE/ad2ad60f-33f1-456e-9fec-7d0c3151f217.pdf",
                "state": null,
                "company": "PTC",
                "city": null,
                "blog": null,
                "firstname": "Dat",
                "facebook": null,
                "is-registered": false,
                "is-checked-in": false,
                "is-checked-out": false,
                "deleted-at": null,
                "fluent-language": null,
                "is-badge-printed": false,
                "github": null,
                "is-consent-form-field-email": false,
                "country": "Vietnam",
                "age-group": null,
                "wiki-scholarship": null,
                "instagram": null,
                "accept-receive-emails": false,
                "device-name-checkin": null,
                "attendee-notes": null,
                "is-consent-form-field-photo": false,
                "accept-share-details": false,
                "gender": null,
                "birth-date": null,
                "work-address": null
            },
            "relationships": {
                "event": {
                    "links": {
                        "self": "/v1/attendees/49414/relationships/event",
                        "related": "/v1/attendees/49414/event"
                    }
                },
                "user": {
                    "links": {
                        "self": "/v1/attendees/49414/relationships/user",
                        "related": "/v1/attendees/49414/user"
                    }
                },
                "ticket": {
                    "links": {
                        "self": "/v1/attendees/49414/relationships/ticket",
                        "related": "/v1/attendees/49414/ticket"
                    }
                }
            },
            "id": "49414",
            "links": {
                "self": "/v1/attendees/49414"
            }
        },
        {
            "type": "attendee",
            "attributes": {
                "accept-video-recording": false,
                "ticket-id": "3458",
                "lastname": "Trinh",
                "checkin-times": null,
                "shipping-address": null,
                "complex-field-values": null,
                "checkout-times": null,
                "twitter": null,
                "tax-business-info": null,
                "website": null,
                "home-address": null,
                "badge-printed-at": null,
                "work-phone": null,
                "billing-address": null,
                "email": "trinhvandat090399@gmail.com",
                "address": null,
                "phone": null,
                "is-consent-of-refund-policy": false,
                "linkedin": null,
                "register-times": null,
                "native-language": null,
                "home-wiki": null,
                "job-title": "Backend",
                "is-consent-form-field": false,
                "pdf-url": "https://api.eventyay.com/generated/tickets/orders/tickets/pdf/ad2ad60f-33f1-456e-9fec-7d0c3151f217/49413/OTJWUHZ6V1/ad2ad60f-33f1-456e-9fec-7d0c3151f217.pdf",
                "state": null,
                "company": "Aibles",
                "city": null,
                "blog": null,
                "firstname": "Leonard",
                "facebook": null,
                "is-registered": false,
                "is-checked-in": false,
                "is-checked-out": false,
                "deleted-at": null,
                "fluent-language": null,
                "is-badge-printed": false,
                "github": null,
                "is-consent-form-field-email": false,
                "country": "Vietnam",
                "age-group": null,
                "wiki-scholarship": null,
                "instagram": null,
                "accept-receive-emails": false,
                "device-name-checkin": null,
                "attendee-notes": null,
                "is-consent-form-field-photo": false,
                "accept-share-details": false,
                "gender": null,
                "birth-date": null,
                "work-address": null
            },
            "relationships": {
                "event": {
                    "links": {
                        "self": "/v1/attendees/49413/relationships/event",
                        "related": "/v1/attendees/49413/event"
                    }
                },
                "user": {
                    "links": {
                        "self": "/v1/attendees/49413/relationships/user",
                        "related": "/v1/attendees/49413/user"
                    }
                },
                "ticket": {
                    "links": {
                        "self": "/v1/attendees/49413/relationships/ticket",
                        "related": "/v1/attendees/49413/ticket"
                    }
                }
            },
            "id": "49413",
            "links": {
                "self": "/v1/attendees/49413"
            }
        }
    ],
    "jsonapi": {
        "version": "1.0"
    }
}

Reply:
![image](https://github.com/fossasia/open-event-server/assets/135117837/dd00093b-2097-4353-a07a-539371a779f8)
It means.
The value of "Order#" will be the value of "identifier" in the JSON above, right?
So the value of "Ticket ID" will be the same as "Order#", right?